dcpquery/db/__init__.py

```diff
@@ -102,10 +103,14 @@ class ProcessFileLink(SQLAlchemyBase):
     __tablename__ = 'process_file_join_table'
     id = Column(Integer, primary_key=True)
     process_uuid = Column(UUID, ForeignKey("processes.process_uuid"))
-    process_file_connection_type = Column('value', Enum(ConnectionTypeEnum))
+    process_file_connection_type = Column(Enum(ConnectionTypeEnum))
     process = relationship(Process)
     file_uuid = Column(UUID)
 
+    __table_args__ = (UniqueConstraint
+                      ('process_uuid', 'process_file_connection_type', 'file_uuid',
```

Comment:
It doesn't look like there needs to be a newline between lines 110-111, should we remove it?

Reply:
👍